tensorflow_federated/python/core/impl/executors/federating_executor.py

```diff
@@ -126,6 +127,320 @@ def type_signature(self):
                                   py_typecheck.type_string(type(self._value))))
 
 
+class IntrinsicStrategy(abc.ABC):
+  @classmethod
+  @abc.abstractmethod
+  def validate_executor_placements(cls, target_executors):
+    pass
+
+
+class DefaultIntrinsicStrategy(IntrinsicStrategy):
```

Comment:
maybe rename this one to `PlaintextIntrinsicStrategy` or something other than `Default`; which one is default could change, and doesn't say much about why this strategy works differently than the others

Reply:
Yeah, this was a hotfix. I felt Plaintext wouldn't make much sense without an accompanying ciphertext/encrypted Strategy, and it doesn't get to the heart of how this differs from e.g. `TrustedAggregationIntrinsicStrategy`. One alternative could be `ParameterServerIntrinsicStrategy` to reflect similarity to that style of distributed training. However this of course references training parameters, which is too specific for how this can be used. Another alternative could be `CentralizedAggregationStrategy`.

Reply:
renamed to `CentralizedAggregationStrategy`